<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="SoundPlotWidget" name="plotWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>330</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Existing Outputs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="outputTree">
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="5" column="1">
            <widget class="QPushButton" name="predictButton">
             <property name="text">
              <string>Predict</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Output:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="playButton">
             <property name="text">
              <string>Play Selection</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>File:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="learnButton">
             <property name="text">
              <string>Learn Selection</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="unlearnButton">
             <property name="text">
              <string>Unlearn</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="outputLineEdit"/>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QPushButton" name="browseButton">
             <property name="text">
              <string>Browse For Wav File</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="loadButton">
             <property name="text">
              <string>Load Memory</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QPushButton" name="clearMemButton">
             <property name="text">
              <string>Clear Memory</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="predictionLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QSpinBox" name="intervalSpin">
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Intervals:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="ConsoleWidget" name="consoleWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="RecorderWidget" name="recorderWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1297</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>Recent Files</string>
     </property>
     <addaction name="actionNo_Recent_Files"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuRecent_Files"/>
    <addaction name="actionSave_Memory"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionNo_Recent_Files">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>No Recent Files</string>
   </property>
  </action>
  <action name="actionSave_Memory">
   <property name="text">
    <string>Save Memory</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConsoleWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.console</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SoundPlotWidget</class>
   <extends>QWidget</extends>
   <header>LearningLetters</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RecorderWidget</class>
   <extends>QWidget</extends>
   <header>realTimeAudio</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
